Implementation of MK docs. (#33)
FAIRmat-NFDI/nyaml: README.md

[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff) ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/nyaml) [![PyPI](https://img.shields.io/pypi/v/nyaml)](https://pypi.org/project/nyaml/) [![Pytest](https://github.com/FAIRmat-NFDI/nyaml/actions/workflows/pytest.yaml/badge.svg)](https://github.com/FAIRmat-NFDI/nyaml/actions/workflows/pytest.yaml)[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1323437.svg)](https://doi.org/10.5281/zenodo.13860810)

# nyaml Tool for NeXus
The `nyaml` tool is a Python package that provides a command line interface for converting NeXus application definitions or base classes from `yaml` file format into the `nxdl.xml` file format and vice-versa. The converter can be called by the command `nyaml2nxdl`.

> Check full documentation of the tool in [gh] https://fairmat-nfdi.github.io/nyaml/

Reviewer: ```suggestion
You can find the full documentation of the tool [here](https://fairmat-nfdi.github.io/nyaml/)
```
gh does not mean anything to new users

Author: Addressed!

8. [Conclusion](#conclusion)

> 9. [References](#references)

Reviewer: please remove this table of contents as well

Author: Addressed at 53d263ed5edd4ec3ca20fb55b07674980f8d137e
